testsuite/integration/clustering/pom.xml

```diff
@@ -708,9 +709,9 @@
             </build>
         </profile>
 
-        <!-- Run cluster tests with standalone-ha.xml profile with Infinispan Server provisioned -->
+        <!-- Run cluster tests with standalone-ha.xml profile with reuseForks == false -->
         <profile>
-            <id>ts.clustering.cluster.ha-infinispan-server.profile</id>
+            <id>ts.clustering.cluster.ha-no-reuse-forks.profile</id>
```

Comment:
If we are renaming this profile, the documentation at `docs/src/main/asciidoc/_testsuite/WildFly_Testsuite_Overview.adoc` should be updated accordingly as well.

Reply:
We discussed this at the time, agreeing on the new name, and it was my understanding that this target was distinguished by running with no reuse forks and not some Infinispan class rule as mentioned in the documentation. If this has changed, i'm not going to rename it. Also, I don't see difference between this target and the full-ha target below it.

Reply:
@rachmatowicz I was just talking about updating the documentation, but you actually raised a more important point!

Yes, what you are describing is what we discussed at that time - basically I suggested to reuse and rename that profile which was using <reuseForks>false</reuseForks>. However, this has changed since, and as of 044c1532d0946a50905bcfddf2dffa755d5dd329 this workaround was removed. 

Looking more closely, you aren't actually putting that <reuseForks>false</reuseForks> back, so your test is passing even with reuse forks enabled! If this is the case, you can just put the test back to the `ts.clustering.cluster.ha` profile. Right?

Reply:
@rachmatowicz So, this might be working because in that execution, there are no other EJB client tests, thus the static EJB context is not present and the tests pass. So if you cannot add this onto other executions, I am going to ask you to create a new one. The `ha-infinispan-server` is now relied on to run the RHDG tests so let's not rename it.

Reply:
It does not work if I include in the ts.clustering.cluster.ha profile. I would prefer not to add another profile just for one test - on the other hand, this test does not play well with other EJB tests and so merits a reuse forks property. Thinking.

Reply:
@rachmatowicz Neither do I, but adding `<reuseForks>false</reuseForks>` to any of the existing test cases increasing testsuite runtime by minutes; the price is managing one extra profile.